app/src/main/java/com/keylesspalace/tusky/components/search/fragments/SearchStatusesFragment.kt

```diff
@@ -303,6 +327,10 @@ class SearchStatusesFragment : SearchFragment<StatusViewData.Concrete>(), Status
             )
         }
 
+        val translateItem = popup.menu.findItem(R.id.status_translate)
+        translateItem.isVisible = status.language != Locale.getDefault().language && viewModel.supportsTranslation()
```

Comment:
Maybe we want to *always* show the translate option when supported?
Rationale: people are bad about tagging their post language

Reply:
that's a good point, I assumed it doesn't work but I think it tries to detect it from text anyway and it's not so annoying in the menu

Reply:
actually I get 403  Forbidden if I try to translate the same language 😐

Reply:
😐